tests/test_api.py

```diff
@@ -1,21 +1,225 @@
-import os
-import chalicelib.lib.dynamo_event as dynamo
 import requests
 
 local_api = 'http://localhost:8010'
 local_db = 'http://localhost:8000'
+testusers = ["testuser1", "testuser2"]
+testmonths = ["2019-03", "2019-04"]
+event_data = [
+    {
+        "reason": "sick",
+        "hours": "8",
+        "user_name": "Test Mctest",
+        "user_id": f"{testusers[0]}",
+        "event_date": "2019-12-30"
+    },{
+        "reason": "sick",
+        "hours": "8",
+        "user_name": "Test Mctest",
+        "user_id": f"{testusers[0]}",
+        "event_date": "2019-12-31"
+    },{
+        "reason": "sick",
+        "hours": "8",
+        "user_name": "Test Mctest",
+        "user_id": f"{testusers[0]}",
+        "event_date": "2020-01-01"
+    }
+]
+lock_data = [
+    {
+        "user_id": f"{testusers[0]}",
+        "event_date": f"{testmonths[0]}"
+    },{
+        "user_id": f"{testusers[0]}",
+        "event_date": f"{testmonths[1]}"
+    }
+]
+
 
 # sanity check to see that we have some certainty that this test will execute
 # on local environment
 check_local_db = requests.get(f"{local_db}/shell/")
 if check_local_db.status_code != 200:
     raise Exception("Local DB check failed")
 
-check_local_api = requests.get(f"{local_api}/table-name")
+check_local_api = requests.get(f"{local_api}/table-names")
 if check_local_api.status_code != 200:
     raise Exception("Local API check failed")
 
-def test_get_id(caplog):
-    response = requests.get(f"{local_api}/event/users/fake_user")
-    assert response is not None
-    assert response.text == '[]'
+
+def create_event(event: dict):
+    r = requests.post(f"{local_api}/events", json=event)
+    return r
+
+
+def create_lock(lock: dict):
+    r = requests.post(f"{local_api}/locks", json=lock)
+    return r
+
+
+def test_create_event():
```

Comment:
All these tests. Might it be valuable to check that the response returned is also valid json?

Reply:
Yes, we should have tests on the data returned. Should have a ticket on that.